lab1.py

```diff
@@ -0,0 +1,59 @@
+
+
+# Task1
+num = int(input("Enter a number: "))
+
+
+flag = False
+
+# prime numbers are greater than 1
+if num > 1:
+
+    for i in range(2, num):
+        if (num % i) == 0:
+
+            flag = True
+
+            break
+
+
+if flag:
+    print(num, "is not a prime number")
+else:
+    print(num, "is a prime number")
+
+# Task2
+
+
+def show_student(name, GPA=0):
+    print("name: "+name)
+    if GPA:
+        print("GPA: "+GPA)
+    else:
+        print(0)
```

Comment:
No need for these lines if you assign a default value to the argument in the function definition `(name, GPA=0)`
So it will take the default value in the definition if you do not pass it.